### PR TITLE
Master volume added 

### PR DESCRIPTION
**Pull request by allocom**
On allo-piano-dac-plus.c: Master volume added, which controls both DACs volume.
On allo-digione.c: 192kHz sampling rate clicking sound fix.

### CODE REVIEW

**Comment by pelwell** [comment]
Although I appreciate the improved adherence to the kernel coding style, try not to mix cosmetic changes (unless very small or non-overlapping) with functional changes - they obscure the real differences. I'd prefer them as separate commits for review, even if they get squashed when merged.

File: sound/soc/bcm/allo-piano-dac-plus.c [hunk 1]
```diff
@@ -474,7 +473,151 @@ static int pcm512x_set_reg_sub_switch(struct snd_kcontrol *kcontrol,
...
+	struct snd_soc_pcm_runtime *rtd;
+	int val = 0;
+
+	rtd = snd_soc_get_pcm_runtime(card, card->dai_link[0].name);
+
+	val = snd_soc_read(rtd->codec_dais[0]->codec, PCM512x_MUTE);
+	if (val < 0)
+		return val;
+
+	ucontrol->value.integer.value[0] = (val & 0x10) ? 0 : 1;
```
Comment (pelwell) [inline sound/soc/bcm/allo-piano-dac-plus.c]: A few more named constants, such as a name for bit 4 of PCM512x_MUTE, would make your code easier to understand.

**Comment by allocom** [comment]
Thanks Pelwell, noted.

**Comment by pelwell** [comment]
Merged manually.

**Pull request closed without merging**